src/components/button.tsx

```diff
@@ -8,21 +8,25 @@ export default function Button(props: {
   ariaLabel?: string;
 }) {
   const classNames = `shadow-sm shadow-wai-gray bg-pure-white text-xl font-mono font-bold text-center text-wai-gray border-4 rounded-lg border-wai-gray p-4 justify-self-center`;
+  
+  // To allow for \n linebreaks
+  const textStyle = { whiteSpace: 'pre-line' };
```

Comment:
we use tailwind for this project, so add `whitespace-pre-line` to the class instead of using `style`

Reply:
Never did get my head round tailwind...